package.json

```diff
@@ -2,8 +2,8 @@
     "name": "cookie-compliance-for-wordpress",
     "version": "2.0.6",
     "engines": {
-        "node": "12.18.3",
-        "npm": "7.5.2"
+        "node": "*",
+        "npm": "*"
```

Comment:
`npm install` complained that we were using unsupported versions, so I changed this.